src/main/java/org/sagebionetworks/web/client/widget/entity/file/ProjectTitleBarViewImpl.java

```diff
@@ -0,0 +1,95 @@
+package org.sagebionetworks.web.client.widget.entity.file;
+
+import com.google.gwt.core.client.GWT;
+import com.google.gwt.uibinder.client.UiBinder;
+import com.google.gwt.uibinder.client.UiField;
+import com.google.gwt.user.client.ui.IsWidget;
+import com.google.gwt.user.client.ui.SimplePanel;
+import com.google.gwt.user.client.ui.Widget;
+import com.google.inject.Inject;
+import org.gwtbootstrap3.client.ui.Heading;
+import org.gwtbootstrap3.client.ui.html.Div;
+import org.sagebionetworks.repo.model.EntityType;
+import org.sagebionetworks.web.client.DisplayUtils;
+import org.sagebionetworks.web.client.widget.EntityTypeIcon;
+
+public class ProjectTitleBarViewImpl implements ProjectTitleBarView {
```

Comment:
The implementation of `ProjectTitleBarViewImpl` is the previous implementation of `BasicTitleBarViewImpl`, with no logical changes--just a rename.